static/app/components/dropdownMenu/index.tsx

```diff
@@ -155,6 +156,7 @@ function DropdownMenu({
   preventOverflowOptions,
   flipOptions,
   portalContainerRef,
+  shouldApplyMinWidth,
```

Comment:
DropdownMenu already accepts a bunch of useOverlay options, this adds one that was missing